src/main/java/oss/fosslight/service/impl/T2UserServiceImpl.java

```diff
@@ -64,608 +60,623 @@
+        authMapper.insertAuthorities(authorityItem);
+      }
+    }
+
+    return 1;
+  }
+
+  @SuppressWarnings("serial")
+  @Override
+  public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+    return new UserDetails() {
+      @Override
+      public boolean isEnabled() {
+        return false;
+      }
+
+      @Override
+      public boolean isCredentialsNonExpired() {
+        return false;
+      }
+
+      @Override
+      public boolean isAccountNonLocked() {
+        return false;
+      }
+
+      @Override
+      public boolean isAccountNonExpired() {
+        return false;
+      }
+
+      @Override
+      public String getUsername() {
+        return null;
+      }
+
+      @Override
+      public String getPassword() {
+        return null;
+      }
+
+      @Override
+      public Collection<? extends GrantedAuthority> getAuthorities() {
+        return null;
+      }
+    };
+  }
+
+  @Override
+  public String getPassword(T2Users user) {
+    T2Users result = userMapper.getPassword(user);
+
+    if (result != null) {
+      return result.getPassword();
+    }
+
+    return null;
+  }
+
+  /**
+   * 유저 목록 조회
+   */
+  @Override
+  public Map<String, Object> getUserList(T2Users t2Users) {
+    int records = userMapper.selectUserTotalCount(t2Users);
+    t2Users.setCurPage(t2Users.getPage());
+    t2Users.setPageListSize(t2Users.getRows());
+    t2Users.setTotListSize(records);
+    List<T2Users> userList = userMapper.selectParamUserList(t2Users);
+
+    HashMap<String, Object> map = new HashMap<String, Object>();
+    map.put("page", t2Users.getPage());
+    map.put("total", t2Users.getTotBlockSize());
+    map.put("records", records);
+    map.put("rows", userList);
+
+    return map;
+  }
+
+  /**
+   * 유저 저장
+   */
+  @Override
+  @Transactional
+  public void setUser(T2Users vo) throws Exception {
+    vo.setCreator(vo.getUserId());
+    vo.setModifier(vo.getUserId());
+
+    userMapper.insertUsers(vo);
+    userMapper.insertAuthority(vo);
+  }
+
+  @Override
+  public void modUser(List<T2Users> vo) {
+    for (T2Users t2Users : vo) {
+      t2Users.setModifier(t2Users.getUserId());
+      t2Users.setPassword("");
+      if (t2Users.getDivision().trim().equals("")) {
+        t2Users.setDivision(CoConstDef.CD_USER_DIVISION_EMPTY);
+      }
+
+      userMapper.updateUsers(t2Users);
+
+      t2Users.setAuthority(t2Users.getAuthority().equals("V") ? "ROLE_ADMIN" : "ROLE_USER");
+      userMapper.updateAuthorities(t2Users);
+
+      // statisticsMostUsed > div_no value update
+      userMapper.updateStatisticsMostUsedInfo(t2Users);
+    }
+  }
+
+  @Override
+  public List<T2Users> getAuthorityUsers(String authority) {
+    return userMapper.getAuthorityUsers(authority);
+  }
+
+  @Override
+  public List<T2Users> getUserListExcel() throws Exception {
+    List<T2Users> result = userMapper.selectUserList();
+
+    for (T2Users t2Users : result) {

```

Comment:
I replaced **for-loop** with **enhanced for-loop** to improve clarity and readability.